<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Scoring</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Import_Page" native="true">
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>210</y>
          <width>391</width>
          <height>248</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="Upload_image_label">
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/image/src/image/upload.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="Upload_label">
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;Consolas&quot;;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Drag &amp;amp; Drop image files &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;or&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="Import_button">
           <property name="minimumSize">
            <size>
             <width>350</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 75 16pt &quot;Consolas&quot;;
</string>
           </property>
           <property name="text">
            <string>Import</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Result_Page" native="true">
       <widget class="QFrame" name="image_result_frame">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>140</y>
          <width>251</width>
          <height>331</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="import_image_label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Image&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="result_table">
           <row>
            <property name="text">
             <string>Stock photo scoring</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>UGC photo scoring</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>Image keyword</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>image 1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="return_button">
        <property name="geometry">
         <rect>
          <x>930</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>return</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Export_button">
        <property name="geometry">
         <rect>
          <x>840</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Export</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="image_rcs.qrc"/>
 </resources>
 <connections/>
</ui>
